src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -192,381 +191,387 @@ the MIT License. See LICENSE in the project root for license information. -->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}"
+                            <!--Background Image Opacity-->
+                            <ContentPresenter Style="{StaticResource SettingContainerStyle}"
                                           Visibility="{x:Bind State.Profile.BackgroundImageSettingsVisible, Mode=OneWay}">
-                            <StackPanel>
-                                <TextBlock x:Uid="Profile_BackgroundImageOpacity"
+                                <StackPanel>
+                                    <TextBlock x:Uid="Profile_BackgroundImageOpacity"
                                            Style="{StaticResource SliderHeaderStyle}"/>
-                                <Grid Style="{StaticResource CustomSliderControlGridStyle}">
-                                    <Grid.ColumnDefinitions>
-                                        <ColumnDefinition Width="*"/>
-                                        <ColumnDefinition Width="Auto"/>
-                                    </Grid.ColumnDefinitions>
-                                    <Slider x:Name="BIOpacitySlider"
+                                    <Grid Style="{StaticResource CustomSliderControlGridStyle}">
+                                        <Grid.ColumnDefinitions>
+                                            <ColumnDefinition Width="*"/>
+                                            <ColumnDefinition Width="Auto"/>
+                                        </Grid.ColumnDefinitions>
+                                        <Slider x:Name="BIOpacitySlider"
                                             Grid.Column="0"
                                             Value="{x:Bind State.Profile.BackgroundImageOpacity, Converter={StaticResource PercentageConverter}, Mode=TwoWay}"/>
-                                    <TextBlock Grid.Column="1"
+                                        <TextBlock Grid.Column="1"
                                                Text="{Binding ElementName=BIOpacitySlider, Path=Value, Mode=OneWay}"
                                                Style="{StaticResource SliderValueLabelStyle}"/>
-                                </Grid>
-                            </StackPanel>
-                        </ContentPresenter>
+                                    </Grid>
+                                </StackPanel>
+                            </ContentPresenter>
+                        </StackPanel>
 
                         <!--Grouping: Acrylic-->
-                        <TextBlock x:Uid="Profile_AcrylicHeader" Style="{StaticResource GroupingHeader}"/>
+                        <StackPanel Style="{StaticResource PivotStackStyle}">
+                            <TextBlock x:Uid="Profile_AcrylicHeader" Style="{StaticResource SubtitleTextBlockStyle}"/>
 
-                        <!--Use Acrylic-->
-                        <ContentPresenter Style="{StaticResource SettingContainerStyle}">

```

Comment:
Again, if you're using a `ContentPresenter` without styling, I suggest styling it, then overriding the margin plz.